### DISCUSSION TITLE
What is this number?

### DISCUSSION THREAD

**Discussion by cr4zyfish** [xjasonlyu/tun2socks#372] [General]
### Version

tun2socks-2.5.2 windows/amd64, go1.20.11, 2b494a7

### What OS are you seeing the problem on?

Windows

### Description

What is this number that increase each time I run tun2socks? am I doing something wrong?
wintun 113 wintun 114 ...

![tun](https://github.com/xjasonlyu/tun2socks/assets/90797624/85560b73-66df-4d03-bddf-babd65dbf0bb)

**Comment by xjasonlyu** [comment]
Hi, wintun will automatically set its adapter id if you didn't specify one, but it's fine. If you want to persist the number, you can use args like `tun://wintun?guid={7FE5A4A1-1326-40E2-974C-EF617156ACA8}` to specify the GUID.

Also see https://github.com/xjasonlyu/tun2socks/issues/300 for reference.
